docs/guide/areas-and-widgets.md

```diff
@@ -166,6 +166,20 @@ The extension should always be lower case.
 
 The second option is `previewIcon`. This option takes any icon that has already been [registered](https://github.com/apostrophecms/apostrophe/blob/main/modules/@apostrophecms/asset/lib/globalIcons.js). Alternatively, additional Material Design Icons can be registered using the [`icons`](https://v3.docs.apostrophecms.org/reference/module-api/module-options.html#icon) property within the module. If it is present, the `icon` option will be used if no `previewIcon` option is set.
 
+## Adding placeholder content to widgets
+
+The rich text, image, and video widgets all display placeholder content by default. Additionally, these modules do not show an initial editing modal. This placeholder content will not be displayed in either the draft preview or the live page if published. This default behavior can be turned off by setting the `placeholder` option to `false`. This will eliminate the display of placeholder content and open the editing modal when the widget is selected.
```

Comment:
... can be turned off by setting the `placeholder` option of the relevant module to `false`

(i.e. this has to be done at module level, it can't be done in area configuration)

Reply:
Changed.